kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -3749,6 +3758,51 @@ messages:
       return;
    }
 
+    UserMoveInventoryItem(what=$,where=$)
```

Comment:
"where" makes it sound like this is the ordinal position to move "what", but it's actually an *object* in the inventory where "what" will move.  Can you rename "where" (maybe to target_object or something better)?

Reply:
Changed parameters to oPayload and oTarget to match what I used in list.c on the client side.  Updated UserMoveInventoryItem function